packages/webapp/src/components/ErrorHandler/PureReactErrorFallback/index.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useTranslation, Trans } from 'react-i18next';
+import styles from './styles.module.scss';
+import { Title, Main } from '../../Typography';
+import TextButton from '../../Form/Button/TextButton';
+import useIsAboveBreakpoint from '../../../hooks/useIsAboveBreakpoint';
+import { ReactComponent as Background } from '../../../assets/images/errorFallback/background.svg';
+import { ReactComponent as MobileBackground } from '../../../assets/images/errorFallback/background_mobile.svg';
+import { ReactComponent as FarmerDesktop } from '../../../assets/images/errorFallback/farmer_desktop.svg';
+import { ReactComponent as FarmerMobile } from '../../../assets/images/errorFallback/farmer_mobile.svg';
+import { ReactComponent as RefreshIcon } from '../../../assets/images/errorFallback/refresh.svg';
+import { ReactComponent as LogoutIcon } from '../../../assets/images/errorFallback/logout.svg';
+import { ReactComponent as Logo } from '../../../assets/images/nav/logo-large.svg';
+import { SUPPORT_EMAIL } from '../../../util/constants';
+
+interface PureReactErrorFallbackProps {
+  handleReload: () => Promise<void>;
+  handleLogout: () => void;
+}
+
+export const PureReactErrorFallback = ({
+  handleReload,
+  handleLogout,
+}: PureReactErrorFallbackProps) => {
+  const { t } = useTranslation();
+
+  const isDesktop = useIsAboveBreakpoint('(min-width: 600px)');
```

Comment:
Lately we've been using MUI's breakpoints and their hook, in this case like this

`const isDesktop = useMediaQuery(theme.breakpoints.down('sm'));`

That way we always use the same breakpoints instead of typing in the number in the components. I think there's only one remaining component using `useIsAboveBreakpoint` which could be refactored as well and then we could remove the custom hook

Reply:
No issues with this component, but for refactoring the last component that uses `useIsAboveBreakpoint`... the breakpoint is `856px`. Is it safe to round up to 900px (MUI breakpoint `md`)? @SayakaOno it is your component so I want to defer to you! This is the floating button on finances.

I think it looks okay at 900 but I want to make sure I'm not missing something 🙂

Reply:
900px looks fine to me too. It looks like it was originally 768px, and I thought I just had followed Loïc's design, but I couldn't find any breakpoints in the mockups... also, it looks like Anto updated it to 856px [here](https://github.com/LiteFarmOrg/LiteFarm/pull/3053/files).
Setting a higher breakpoint seems safe...